logicrunner/execution_archive.go

```diff
@@ -122,10 +122,14 @@ func (ea *executionArchive) OnPulse(_ context.Context) []insolar.Message {
 	return messages
 }
 
-func (ea *executionArchive) FindRequestLoop(ctx context.Context, apiRequestID string) bool {
+func (ea *executionArchive) FindRequestLoop(ctx context.Context, reqRef insolar.Reference, apiRequestID string) bool {
 	ea.archiveLock.Lock()
 	defer ea.archiveLock.Unlock()
 
+	if _, ok := ea.archive[reqRef]; ok {
+		return false
```

Comment:
Could you please write a little comment here that would explain this code?

Reply:
added comment